buf.yaml

```diff
@@ -0,0 +1,19 @@
+build:
+  roots:
+    - .
+  excludes:
+    - vendor/github.com/tendermint
+lint:
+  use:
+    - DEFAULT
+    - COMMENTS
+    - FILE_LOWER_SNAKE_CASE
+  except:
+    - UNARY_RPC
```

Comment:
Do we have any use case for streaming RPC?

Reply:
No, we don't atm.